src/driver/include/driver/DefaultDriver.hpp

```diff
@@ -28,8 +28,24 @@ namespace genny::driver {
  * Basic workload driver that spins up one thread per actor.
  */
 class DefaultDriver {
-
 public:
+    enum class RunMode {
+        kNormal = 0,
+        kDryRun = 1,
+        kEvaluate = 2,
+        kListActors = 3,
+        kSmokeTest = 4,
```

Comment:
Unused? I know Ryan mentioned it in the ticket description but it seems like when we add support for it in code we can the enum case.

Reply:
SG. Removed